pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, scsh, feh }:
+stdenv.mkDerivation rec {
+  pname = "deco";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "ebzzry";
+    repo = pname;
+    rev = "f58e931ccea359579a9192748d1085fae73a9d47";
+    sha256 = "06zdb0wgxjf1xk7d0lsc82dmx2vz0a7mn1vkn79x0inyfgp4bdql";
+  };
+
+  propagatedBuildInputs = [ scsh feh ];
```

Comment:
I recommend patching in store paths instead of doing this, in this case.

Having briefly looked at the source, I see there are a couple other utilities that are being called with the assumption that they'll be in PATH.